examples/vscode-sandbox/README.md

```diff
@@ -0,0 +1,97 @@
+
+This example uses a ResourceGraphDefinition (RGD) to define an DevContainer CRD.
+For more details on RGD please look at [KRO Overview](https://kro.run/docs/overview)
+
+## Install KRO
+
+Follow instructions to [Install KRO](https://kro.run/docs/getting-started/Installation)
+
+## Administrator: Install ResourceGraphDefinition
+The administrator installs the RGD in the cluster first before the user can consume it:
+
+```
+kubectl apply -f rgd.yaml
+```
+
+Validate the RGD is installed correctly:
+
+```
+kubectl get rgd
+
+NAME            APIVERSION   KIND             STATE    AGE
+devcontainer    v1alpha1     DevContainer   Active   6m38s
+```
+
+Validate that the new CRD is installed correctly
+```
+kubectl get crd
+
+NAME                                   CREATED AT
+devcontainers.custom.agents.x-k8s.io   2025-09-20T05:03:49Z  # << THIS
+resourcegraphdefinitions.kro.run       2025-09-20T04:35:37Z
+sandboxes.agents.x-k8s.io              2025-09-19T22:40:05Z
+```
+
+## User: Create DevContainer
+
+The user creates a `DevContainer` resource something like this:
+
+```
+kubectl apply -f instance.yaml
+```
+
+```yaml
+apiVersion: custom.agents.x-k8s.io/v1alpha1
+kind: DevContainer
+metadata:
+  name: demo
+spec:
+  giturl: https://github.com/barney-s/agent-sandbox.git#refs/heads/devcontainer
```

Comment:
Nit: gitUrl (or `spec.source.git` ?)

Reply:
moving this to KRO. specifically calling this giturl because we parse it as a git-url and extract the branch from the url if needed. source.gitURL maybe. But we only support one source.

Reply:
changed to .source.giturl